<launch>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find whole_body_control)/config/wbc.rviz"/>
    <!-- 将 Urdf 文件的内容加载到参数服务器 -->
    <param name="robot_description" command="$(find xacro)/xacro $(find whole_body_control)/urdf/arm.urdf.xacro" />
    <!-- 启动 gazebo -->
    <!-- <include file="$(find gazebo_ros)/launch/empty_world.launch" /> -->

    <!-- 在 gazebo 中显示机器人模型 -->
    <!-- <node pkg="gazebo_ros" type="spawn_model" name="model" args="-urdf -model mycar -param robot_description"  /> -->

    <!-- <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" /> -->
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
    <node name="wbc_node" pkg="whole_body_control" type="wbc.py" output="screen"/>
    <node name="arm_cycle_controller" pkg="whole_body_control" type="base_motion.py" output="screen"/>
</launch>